packages/react/src/feature-link-list/FeatureLinkList.test.tsx

```diff
@@ -0,0 +1,72 @@
+import '@testing-library/jest-dom';
+import 'html-validate/jest';
+import { axe, toHaveNoViolations } from 'jest-axe';
+import { render, cleanup, screen } from '../../../../test-utils';
+import { FeatureLinkList, FeatureLinkListProps } from './FeatureLinkList';
+
+expect.extend(toHaveNoViolations);
+
+afterEach(cleanup);
+
+function renderLinkList(props?: FeatureLinkListProps) {
+	const { links, ...restProps } = props ?? {};
+	return render(
+		<FeatureLinkList
+			links={
+				links ?? [
```

Comment:
Links is mandatory, so this feels unnecessary

Reply:
I think what @stowball is getting at is this default set of links should be created earlier in the function so that it's clearer that's what they're for. Also, if the props arg is `Partial<FeatureLinkListProps>` then you can selectively override just the optional props when writing tests.